gen/flights/flight.py

```diff
@@ -175,7 +176,7 @@ def points(self) -> List[FlightWaypoint]:
         return self.flight_plan.waypoints[1:]
 
     def __repr__(self):
-        name = db.unit_type_name(self.unit_type)
+        name = db.unit_pretty_name(self.country, self.unit_type)
```

Comment:
`__repr__` should use the ID. It's the debug representation. `__str__` is for pretty printing.

Reply:
Ah, good to know - thanks! Fixed.